### PR TITLE
release 0.4.0 rc0

### PR DESCRIPTION
**Pull request by justin1121**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.4.0]
+
+**Added**
+
+- SecureNN with int64 support has landed.
```
Comment (mortendahl) [inline CHANGELOG.md]: maybe mark somewhere that a nightly/custom version of TF is needed for int64 (but CRT works in either case)

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Cryptographically secure random numbers feature has been implemented but not integrated.
```
Comment (mortendahl) [inline CHANGELOG.md]: maybe it's worth switching to the un-seeded secure random ops before releasing? would it require a lot? then we could avoid the "but not integrated" part

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Three new ops have been added to Pond and the converter: Pad, BatchToSpaceND, SpaceToBatchND
+
+**Changed**
+
+- There are now separate wheels published to pypi for MacOS and linux.
```
Comment (ianlivingstone) [inline CHANGELOG.md]:
```suggestion
- There are now separate wheels published to PyPI for MacOS and Linux.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.4.0]
+
+**Added**
+
+- SecureNN with int64 support has landed.
+- Cryptographically secure random numbers feature has been implemented but not integrated.
+- Three new ops have been added to Pond and the converter: Pad, BatchToSpaceND, SpaceToBatchND
```
Comment (ianlivingstone) [inline CHANGELOG.md]: Perhaps just mention we now support three new operations?

File: CHANGELOG.md (hunk 1 above)
Comment (justin1121) [reply CHANGELOG.md]: O yeah. Thanks.

File: CHANGELOG.md (hunk 2 above)
Comment (justin1121) [reply CHANGELOG.md]: Sure lets do it. And then we can remove the part from the README too.

**Pull request merged**